.github/dependabot.yml

```diff
@@ -0,0 +1,34 @@
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 10
+    allow:
```

Comment:
I used dependabot in the past and was working well, but I think in an "extension" repo this can be tricky. Similar to @ankur22 comment [here](https://github.com/grafana/xk6-browser/pull/1051#pullrequestreview-1646158314), I think we should add to the allowed list only the dependencies that are directly related with xk6-browser, for the rest, the ones that are used in k6 and transitively in k6 browser, I would not add them, as we will always be "restricted" by k6 core updates pace.

What I would make sure to include is every security related update, so at least we are aware. Not sure if this is already included in this definition :thinking:

Reply:
I agree; that's why I [was hesitant](https://github.com/grafana/xk6-browser/pull/1051#discussion_r1338421455) to add Goja. Yes, the security updates should be included. And, as I [explained earlier](https://github.com/grafana/xk6-browser/pull/1051#issuecomment-1737160204), we can manage the bot by [its UI](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/managing-pull-requests-for-dependency-updates#managing-dependabot-pull-requests-with-comment-commands). So it won't be like a fully automated progress, and we don't have to include everything the bot says. We can fine-tune this config in time. Let's first see how it will work for us.

Reply:
So should we remove the goja dependency then? I think it will add just noise for us to review if we are just going to do whatever k6 does, and also that repo is pretty active in development.

Reply:
As I said, we can tune the bot in time :) But, sure, we can do that 👍 @ankur22, WDYT?

Reply:
Yeah, sounds good 👍

Should the other shared dependencies also be removed such as `github.com/stretchr/testify` and `github.com/mailru/easyjson`?

Reply:
I think we can keep `testify` and remove `easyjson`. `testify` is fine because we never merge tests into k6-core.

Reply:
See [b2c72c4](https://github.com/grafana/xk6-browser/pull/1051/commits/b2c72c44d583154027e4444f9943adc65deb9f69) and [8e28d45](https://github.com/grafana/xk6-browser/pull/1051/commits/8e28d4525801d524098f783d4d6b79b0730aabd1).